modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/rest-openapi.yaml

```diff
@@ -940,7 +940,7 @@ paths:
                       format: date-time
                       type: string
                 - in: query
-                  name: processStatuses
+                  name: instanceStatuses
```

Comment:
Hey @MarinhoFeliphe, The `instanceStatuses` seems a little bit redundant. Maybe we can use just `statuses`. What do you think?

Thx